Moya.podspec

```diff
@@ -31,7 +31,7 @@ Pod::Spec.new do |s|
   s.subspec "ReactiveCocoa" do |ss|
     ss.source_files = "Moya/ReactiveCocoa/*.swift"
     ss.dependency "Moya/ReactiveCore"
-    ss.dependency "ReactiveCocoa", "~> 4.0-alpha.1"
+    ss.dependency "ReactiveCocoa", "= 4.0.0-alpha-3"
```

Comment:
I don't think the pod file needs to specify a specific version – using the `~> 4.0-alpha.1` syntax will let people pick whichever alpha they want. I think this specifier should be added to the Demo project's Podfile, though.

```rb
pod 'ReactiveCocoa', '~> 4.0-alpha.1'
```